### ISSUE TITLE
The windows are frozen after exiting immersive mode from webxr/vr videos after turning on/off WebXR from settings while the videos are opened

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3378]
**[Affected Versions]:**
- Firefox Reality version 10 RC5 (e84d3c92)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- Pico Neo2 - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality browser is open.

**[Steps to reproduce]:**
1. Open in 3 different windows this 3 videos: [video1](https://mixedreality.mozilla.org/hello-webxr/), [video2](https://krpano.com/releases/1.20.6/viewer/krpano.html?xml=examples/demotour-apartment/tour.xml), [video3](https://www.360cities.net/image/mini-tenerife).
2. Open the Privacy and Security card from Settings.
3. Turn Off and On WebXR (on Quest I had to do it twice).
4. Dismiss the prompts and enter/exit immersive mode in the opened videos one by one multiple times.
5. Observe the behavior.

**[Expected results]:**
- The videos are successfully navigated.

**[Actual results]:**
- Most of the times the windows are frozen and once the content crashed.

**[Notes]:**
- The issue is only reproducible if the videos are entered/exited immersive mode multiple times.
- In some attempts the "Graphic content lost" message was displayed for the videos that use webvr, after exiting immersive mode. 
- The issue is only reproducible after exiting immersive mode.
- After the windows are frozen, if the app is quited and resumed a black screen is displayed.
- I will verify tomorrow if the issue is also reproducible on the HTC device.
- Attached are screen recordings and the crash report of the issue:
	- Screen Recording Quest: [link](https://drive.google.com/open?id=1SSy7L-C03GuID98DgCYEAT9QaXoA71Cc).
	- Screen Recording Pico Neo2: [link](https://drive.google.com/open?id=1YUyht0iF0lQjBukYYjD1OeOhwnDuvlUM).
	- Crash Report Pico Neo2: [link](https://crash-stats.mozilla.org/report/index/d633578f-bcc6-45fc-b787-bef020200514).

**Comment by Softvision-RemusDranca** [comment]
I've retested this issue and I was still able to reproduce it using Firefox Reality RC6 (140fb70) but only after turning WebXR Off/ON multiple times from settings.
Now, the content froze while trying to enter immersive mode in the 360cities example and when I tried to close one of the windows, the infinite WebXR spinner was displayed. 
Tested on Oculus Quest and Pico Neo2.
- Attached is a screen recording of this behavior: [link](https://drive.google.com/open?id=1IqSgyyIi_-HeWnwfEF8xwxPgwyx1RSjt).

Considering this I will reopen this issue.
